ic-http-mock/example_canister/src/canister_backend/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "canister_backend"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+candid = "0.8.2"
+ic-cdk = "0.6.0"
+ic-cdk-macros = "0.6.0"
+ic-http-mock = {path = "../../../"}
+serde = { version = "1.0.158", features = [ "derive" ] }
+serde_json = "1.0.94"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+tokio = { version = "1.15.0", features = [ "full" ] }
+
+[dev-dependencies]
+tokio = { version = "1.15.0", features = [ "full" ] }
```

Comment:
Are you sure this is needed? The dependency is already declared as a non-dev dependency, and we can't include tokio I believe in the wasm build, which would make this redundant.

Reply:
Done.